test/integration/cli/create-rm.bats

```diff
@@ -52,10 +52,10 @@ load ${BASE_TEST_DIR}/helpers.bash
   [[ ${lines[0]} == "Error creating machine: Machine A already exists" ]]
 }
 
-@test "none: extraneous argument is ignored in name 'machine create -d none --url none a foo'" {
-  run machine create -d none a foo
+@test "none: fail with trailing options 'machine create -d none --url none a foo'" {
+  run machine create -d none --url none a foo
   [ "$status" -eq 1 ]
-  [[ ${lines[0]} == "Host already exists: \"a\"" ]]
+  [[ ${lines[0]} == "Invalid command line. All options should be either before or after the machine name. Found those trailing flags [foo]" ]]
```

Comment:
This error message doesn't make much sense to me as `foo` is not a flag, it's an extra argument.